### PR TITLE
Add upload config to appserver chart

### PR DESCRIPTION
**Pull request by pvannierop**
This PR adds helm chart configuration for the new file upload endpoint of appserver.

### CODE REVIEW

**Review by keyvaann — COMMENTED** [review]
Added some comments

File: charts/radar-appserver/values.yaml [hunk 1]
```diff
@@ -234,3 +234,26 @@ smtp:
...
+
+upload:
+  # -- if set to true, file upload endpoint will be enabled
+  enabled: false
+  # -- Maximum file size for upload
+  max_file_size: "20MB"
+  # Settings for file storage
+  storage:
+    # -- Type of storage to use for file upload (s3)
+    type: s3
```
[the remark this answers is not in the document]
Comment (pvannierop) [reply charts/radar-appserver/values.yaml:246]: No other, only _s3_ at this time.

File: charts/radar-appserver/values.yaml [hunk 2]
```diff
@@ -234,3 +234,26 @@ smtp:
...
+    # -- Internal url to storage
+    url: http://minio-headless:9000
```
[the remark this answers is not in the document]
Comment (pvannierop) [reply charts/radar-appserver/values.yaml:248]: It is not yet implemented in the application. Should I add it to the helm chart already? Indeed _region_ is missing and this explains the problems using AWS S3.

**Review by keyvaann — APPROVED** [review]
LGTM

**Pull request merged**